### PR TITLE
Rel 3882: 100 W QH Lamp support to seqexec.

### PR DESCRIPTION
**Pull request by cquiroz**
Can we release this right away or do we need to wait for the next odb release?
@jluhrs Please check the epics channels?

### CODE REVIEW

File: modules/server/src/main/scala/seqexec/server/gcal/GcalControllerEpics.scala [hunk 1]
```diff
@@ -67,9 +67,13 @@ object GcalControllerEpics {
...
 
-  private def setQHLampParams[F[_]: Async](sys: GcalEpics[F])(v: BinaryOnOff): F[Unit] =
-    sys.lampsCmd.setQHLampName("QH") *>
-      sys.lampsCmd.setQHLampOn(v)
+  private def setQH5WLampParams[F[_]: Async](sys: GcalEpics[F])(v: BinaryOnOff): F[Unit] =
+    sys.lampsCmd.setQH5WLampName("QH") *>
+      sys.lampsCmd.setQH5WLampOn(v)
+
+  private def setQH100WLampParams[F[_]: Async](sys: GcalEpics[F])(v: BinaryOnOff): F[Unit] =
+    sys.lampsCmd.setQH100WLampName("QH") *>
```
Comment (cquiroz) [inline modules/server/src/main/scala/seqexec/server/gcal/GcalControllerEpics.scala:75]: This seems suspicious

File: modules/server/src/main/resources/Gcal.xml [hunk 2]
```diff
@@ -50,10 +55,15 @@
...
+            <parameter name="nameQH5W">
                 <channel>lampSel.C</channel>
                 <type>STRING</type>
-                <description>QH Lamp name</description>
+                <description>QH 5W Lamp name</description>
+            </parameter>
+            <parameter name="nameQH100W">
+                <channel>lampSel.N</channel>
+                <type>STRING</type>
+                <description>QH 100W Lamp name</description>
```
Comment (jluhrs) [inline modules/server/src/main/resources/Gcal.xml:66]: This must be lampSel.M

File: modules/server/src/main/scala/seqexec/server/gcal/GcalControllerEpics.scala (hunk 1 above)
Comment (jluhrs) [reply modules/server/src/main/scala/seqexec/server/gcal/GcalControllerEpics.scala:75]: It must be "QH100"

**Review by jluhrs — APPROVED** [review]
LGTM

**Pull request merged**